doc/conf.py

```diff
@@ -18,6 +18,8 @@
 import os, sys
 from datetime import datetime
 
+version = "2.0.0"
```

Comment:
Because (maybe foolishly) I wanted to use namespace packages, this being 'luma.oled', the core one being 'luma.core', and so on, with the 'luma' part being the namespace. Trying to refer to something inside the namespaced object didnt work.